### PR TITLE
Change `Int.MaxValue` to 8 for maximum concurrency during loading in …

### PR DESCRIPTION
**Pull request by unterstein**
…DeploymentRepository.

Summary:
As discussed in #5676 and #5659, we should introduce a restriction on the amount of simultaneous calls. Therefore we change Int.MaxValue to 8 for maximum concurrency during DeploymentRepository.

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala [hunk 1]
```diff
@@ -136,7 +136,7 @@ class DeploymentRepositoryImpl[K, C, S](
   override def ids(): Source[String, NotUsed] = repo.ids()
 
   override def all(): Source[DeploymentPlan, NotUsed] =
-    repo.ids().mapAsync(Int.MaxValue)(get).collect { case Some(g) => g }
+    repo.ids().mapAsync(8)(get).collect { case Some(g) => g }
```
Comment (jeschkies) [inline src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala]: Why is it a literal here but not in #5676?

Comment (ichernetsky) [reply src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala]: 👍  Could you please introduce a const instead and use it here?

Comment (unterstein) [reply src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala]: introduced `RepositoryConstants.maxConcurrency`

File: src/main/scala/mesosphere/marathon/core/storage/repository/Repository.scala [hunk 2]
```diff
@@ -49,3 +49,7 @@ trait VersionedRepository[Id, T] extends ReadOnlyVersionedRepository[Id, T] with
   // Removes _only_ the current value, leaving all history in place.
   def deleteCurrent(id: Id): Future[Done]
 }
+
+object RepositoryConstants {
+  val maxConcurrency = 8
+}
```
Comment (ichernetsky) [inline src/main/scala/mesosphere/marathon/core/storage/repository/Repository.scala]: Could you please add a new line here?

File: src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala [hunk 3]
```diff
@@ -8,17 +8,18 @@ import akka.http.scaladsl.marshalling.Marshaller
...
-import mesosphere.marathon.core.storage.store.{ IdResolver, PersistenceStore }
-import mesosphere.marathon.state.{ RootGroup, Timestamp }
-import mesosphere.marathon.storage.repository.GcActor.{ StoreApp, StorePlan, StorePod, StoreRoot }
+import mesosphere.marathon.core.storage.store.{IdResolver, PersistenceStore}
+import mesosphere.marathon.state.{RootGroup, Timestamp}
+import mesosphere.marathon.storage.repository.GcActor.{StoreApp, StorePlan, StorePod, StoreRoot}
 
-import scala.async.Async.{ async, await }
-import scala.concurrent.{ ExecutionContext, Future, Promise }
+import scala.async.Async.{async, await}
```
Comment (ichernetsky) [inline src/main/scala/mesosphere/marathon/storage/repository/DeploymentRepositoryImpl.scala]: Could you please re-format the imports?

**Comment by unterstein** [comment]
Addressed your feedback @ichernetsky PTAL

**Pull request merged**